user-guide/Troubleshooting/Procedures/Investigating_OpenSearch_Issues.md

```diff
@@ -0,0 +1,165 @@
+          at org.opensearch.transport.RequestHandlerRegistry.processMessageReceived(RequestHandlerRegistry.java:106) ~[opensearch-2.8.0.jar:2.8.0]
+          at org.opensearch.transport.InboundHandler$RequestHandler.doRun(InboundHandler.java:453) ~[opensearch-2.8.0.jar:2.8.0]
+          at org.opensearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:806) ~[opensearch-2.8.0.jar:2.8.0]
+          at org.opensearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:52) ~[opensearch-2.8.0.jar:2.8.0]
+          at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
+          at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
+          at java.lang.Thread.run(Thread.java:833) ~[?:?]
+  ```
+
+To fix this issue:
+
+1. Stop OpenSearch on the cluster manager and data nodes.
+
+1. Check the *cluster.initial_master_nodes* and *discovery.seed_hosts* on all nodes for any differences.
+
+1. Go to the data folder specified in the *opensearch.yml* file and delete the folder called *nodes* along with every file in it by using the following command:
+
+   ```bash
+   sudo rm -rf nodes
+   ```
+
+1. Restart OpenSearch on the cluster manager first and then on the data nodes.
+
+> [!IMPORTANT]
+> Deleting the nodes folder will result in loss of data. You should only do so with a new installation of OpenSearch.
+
+## SLSearch.txt logging mentions OpenSearch version is not officially supported
+
+If the *SLSearch.txt* log file mentions that the OpenSearch version is not officially supported, you can resolve this by upgrading your DMS to DataMiner 10.3.6 or higher.
+
+However, note that this has no functional impact, as the DMA will run fine even if you have not upgraded yet.
+
+## OpenSearch service going into timeout
+
+It can occur that you get an OpenSearch service timeout when executing one of the following commands:
+
+```bash
+sudo systemctl start opensearch
+```
+
+```bash
+sudo systemctl restart opensearch
+```
+
+For example:
+
+```txt
+opensearch.service: start operation timed out. Terminating.
+opensearch.service: Failed with result 'timeout'.
+Failed to start OpenSearch.
+opensearch.service: Consumed 57.702s CPU time.
+```
+
+To resolve this, you may need to increase the start timeout for systemd (see [systemd](https://systemd.io/)):
+
+1. Open the configuration using Nano editor.
+
+   ```bash
+   sudo nano /usr/lib/systemd/system/opensearch.service
+   ```
+
+1. Increase the value of *TimeoutStartSec* to a higher value, for example 300.
+
+   ```txt
+   TimeoutStartSec=300
+   ```
+
+1. Enable the OpenSearch service again:
+
+   ```bash
+   sudo /bin/systemctl enable opensearch.service
+   ```
+
+1. Start the OpenSearch service again:
+
+   ```bash
+   sudo systemctl start opensearch
+   ```
+
+1. Execute the following command to verify that OpenSearch no longer goes into timeout:
```

Comment:
This only verifies whether the opensearch service is able to remain running and does not stop, not really related to timout.

Reply:
Though in the scope of the procedure it is the correct thing to check, maybe we should expand a bit to say this is just to know if the service is running and remains so.